packages/flakybot/src/config-schema.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "flakybot config",
+    "description": "Schema for flakybot configuration",
+    "additionalProperties": false,
+    "properties": {
+	"defaultIssuePriority": {
```

Comment:
Should this just be `issuePriority`? This is the only config, so this isn't really the default, right?

Reply:
Thanks, I hesitant to use `issuePriority` because I'm also planning to introduce dynamic priority, but default sounds incorrect.
I'll change it to `issuePriority`.